Applications/Spire/Include/Spire/BookView/BookViewPropertiesWindowFactory.hpp

```diff
@@ -0,0 +1,45 @@
+#ifndef SPIRE_BOOK_VIEW_PROPERTIES_WINDOW_FACTORY_HPP
+#define SPIRE_BOOK_VIEW_PROPERTIES_WINDOW_FACTORY_HPP
+#include "Spire/BookView/BookViewPropertiesWindow.hpp"
+
+namespace Spire {
+
+  /** A factory used to create the BookViewPropertiesWindow. */
+  class BookViewPropertiesWindowFactory {
+    public:
+
+      /**
+       * Constructs a BookViewPropertiesWindowFactory using the default
+       * properties.
+       */
+      BookViewPropertiesWindowFactory();
+
+      /**
+       * Constructs a BookViewPropertiesWindowFactory.
+       * @param properties The properties model of the book view.
+       */
+      explicit BookViewPropertiesWindowFactory(
+        std::shared_ptr<BookViewPropertiesModel> properties);
+
+      /** Returns the properties displayed. */
+      const std::shared_ptr<BookViewPropertiesModel>& get_properties() const;
+
+      /*
+       * Returns the newly created BookViewPropertiesWindow.
```

Comment:
Does this make a new window? The word new is a term of art that has a specific meaning in C++, but this factory looks like it reuses values.